<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReplaceImageControlForm</class>
 <widget class="QWidget" name="ReplaceImageControlForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>171</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>色置換</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>171</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>分離</string>
   </property>
   <widget class="QRadioButton" name="radioButtonIsolateByColor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>背景の色抽出</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonIsolateByMask">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>マスク画像</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonIsolateByOutline">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>輪郭抽出</string>
    </property>
   </widget>
   <widget class="QFrame" name="frameIsolateByColor">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="widgetIsolateByColor" native="true">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>0</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>240</red>
           <green>240</green>
           <blue>240</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButtonIsolateByColor">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>背景色抽出</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameIsolateByMask">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButtonLoadMaskImage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>マスク画像読込</string>
     </property>
    </widget>
   </widget>
   <widget class="QRadioButton" name="radioButtonIsolateByDrawing">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>描画領域</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>171</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>合成</string>
   </property>
   <widget class="QRadioButton" name="radioButtonCompositeImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>置換画像</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonTransparent">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>透過色</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonCompositePage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>別カメラ映像/画像</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCompositeInside">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>内側</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonViewMask">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>マスク領域（緑色）</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonCompositeColor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>色置換</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frameCompositeColor">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="widgetCompositeColor" native="true">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>0</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>240</red>
           <green>240</green>
           <blue>240</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonCompositeColor">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>設定色</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameCompositeImage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButtonLoadCompositeImage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>置換画像読込</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameCompositePage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="comboBoxCompositePage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>カメラ２</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>カメラ３</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>カメラ４</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QRadioButton" name="radioButtonCompositeNone">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>合成無し</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>radioButtonIsolateByColor</tabstop>
  <tabstop>radioButtonIsolateByMask</tabstop>
  <tabstop>radioButtonIsolateByOutline</tabstop>
  <tabstop>checkBoxCompositeInside</tabstop>
  <tabstop>radioButtonViewMask</tabstop>
  <tabstop>radioButtonCompositeNone</tabstop>
  <tabstop>radioButtonCompositeColor</tabstop>
  <tabstop>radioButtonTransparent</tabstop>
  <tabstop>radioButtonCompositeImage</tabstop>
  <tabstop>radioButtonCompositePage</tabstop>
  <tabstop>toolButtonIsolateByColor</tabstop>
  <tabstop>pushButtonLoadMaskImage</tabstop>
  <tabstop>pushButtonCompositeColor</tabstop>
  <tabstop>pushButtonLoadCompositeImage</tabstop>
  <tabstop>comboBoxCompositePage</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
